core/text/reformatters.talon-list

```diff
@@ -0,0 +1,5 @@
+list: user.reformatters
```

Comment:
I realize you didn't name this but I just noticed it. While we're not consistent (…) most lists are named with a singular rather than plural noun. I don't think we need to worry about backwards compatibility here since reformatters are so new and I couldn't find any references on search.talonvoice.com aside from copies of community.